modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -155,12 +155,20 @@ private void _addNestedFields(
 				objectRelationship.getObjectDefinitionId1());
 
 		if (objectDefinition.isSystem()) {
-			value = _toDTO(
-				objectDefinition.getCompanyId(), primaryKey,
-				_systemObjectDefinitionMetadataRegistry.
-					getSystemObjectDefinitionMetadata(
-						objectDefinition.getName()),
-				dtoConverterContext.getUser());
+			if (GetterUtil.getBoolean(
+					PropsUtil.get("feature.flag.LPS-172094"))) {
+
+				value = _objectEntryLocalService.getSystemModelAttributes(
```

Comment:
This seems strange to me. I am OK to keep old functionality under a feature flag.
But I think that there is a little mistake here, we should return the DTO when the FF is activated and otherwise return the `SystemModelAttributes` but we are doing the opposite. 
Could you please change it?

Reply:
My bad, I am mistaken

Reply:
I just realized that we want to do a breaking change intentionally and, if someone is affected, they should use the FF, so this code is correct. Sorry for the confusion 😓